tools/wpt/run.py

```diff
@@ -559,7 +574,7 @@ def run(venv, **kwargs):
                              install_browser=install_browser,
                              **kwargs)
 
-    rv = run_single(venv, **kwargs) > 0
+    rv = run_single(venv, **kwargs) > 0 or has_log.has_log
```

Comment:
Perhaps stupidly, this does actually run everything (esp. if the CRITICAL error is in setup) and then exit non-zero. But I don't really see how to make a logger call cause a quick exit? Maybe we should just go through every CRITICAL case in the codebase and make it throw/return after?